src/sentry/models/groupresolution.py

```diff
@@ -56,6 +56,18 @@ def has_resolution(cls, group, release):
 
         This is used to suggest if a regression has occurred.
         """
+
+        def compare_release_dates_for_in_next_release(res_release, res_release_datetime, release):
+            """
+            Helper function that compares release versions based on date for
+            `GroupResolution.Type.in_next_release`
+            """
+            if res_release == release.id:
+                return True
+            elif res_release_datetime > release.date_added:
+                return True
+            return False
```

Comment:
Nit: `return res_release == release.id or res_release_datetime > release.date_added` (unless we think this is likely to handle more cases)

Reply:
Nice!